src/gui/qgscolorbutton.cpp

```diff
@@ -420,6 +420,21 @@ void QgsColorButton::dropEvent( QDropEvent *e )
   }
 }
 
+void QgsColorButton::wheelEvent( QWheelEvent *event )
+{
+  if ( mAllowOpacity && isEnabled() && !isNull() )
+  {
+    const int increment = ( ( event->modifiers() & Qt::ControlModifier ) ? 80 : 25 ) *
+                          ( event->angleDelta().y() > 0 ? 1 : -1 );
+    const int alpha = std::min( std::max( 0, mColor.alpha() + increment ), 255 );
+    mColor.setAlpha( alpha );
```

Comment:
I think we should get into the habit of using doubles for opacity (and probably other color components) and using things like setAlphaF here instead. Otherwise there's a loss of precision because we're rounding to just 8 bit color depth.